src/urllib3/__init__.py

```diff
@@ -30,17 +30,19 @@
 except ImportError:
     pass
 else:
-    # fmt: off
-    if (
-        not ssl.OPENSSL_VERSION.startswith("OpenSSL ")
-        or ssl.OPENSSL_VERSION_INFO < (1, 1, 1)
-    ):  # Defensive:
+    if not ssl.OPENSSL_VERSION.startswith("OpenSSL "):  # Defensive
+        warnings.warn(
+            "urllib3 v2.0 only supports OpenSSL 1.1.1+, currently "
+            f"the 'ssl' module is compiled with {ssl.OPENSSL_VERSION}. "
```

Comment:
Let's add a `!r` suffix to this format

Reply:
Done in [`47cce87` (#3024)](https://github.com/urllib3/urllib3/pull/3024/commits/47cce8742b4c5722f6bde4c5de984fcfdfb67f4c) for both messages